test/powershell/Modules/Microsoft.PowerShell.Core/Pester.Commands.Cmdlets.GetCommand.Tests.ps1

```diff
@@ -10,8 +10,8 @@ Describe "Tests Get-Command with relative paths and wildcards" -Tag "CI" {
         #$null = New-Item -ItemType File -Path (Join-Path $TestDrive WildCardCommandA.exe) -ErrorAction Ignore
         #$null = New-Item -ItemType File -Path (Join-Path $TestDRive WildCardCommand[B].exe) -ErrorAction Ignore
         if ( $IsLinux -or $IsMacOS ) {
-            /bin/chmod 777 "$file1"
-            /bin/chmod 777 "$file2"
+            /bin/chmod u+x "$file1"
```

Comment:
I'm not sure whether `Setup` runs as the current user, but if it does, this change should be ok. As this is test code, I would suggest that MSRC is being a bit too sensitive. I'm not positive about how Pester removes files, and relying on umask to set the permissions appropriately is not a good idea. I expect the whole purpose of this setting is to ensure that the file can be deleted, execution perms may not be needed at all.

Reply:
```suggestion
            /bin/chmod a+rw "$file1"
```